.env.sample

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL="SOME_URL"
```

Comment:
No reason to have this file be hidden. Also, add the newline at the end to remove this warning from Github